### PR TITLE
Consolidate system and configuration requirements into one spot.

### PR DESCRIPTION
**Pull request by slack**
* Include version recommendation for Kubernetes
* Include version recommendation for Docker daemon
* Document insecure registry requirement

### CODE REVIEW

File: src/installing-deis/quickstart.md [hunk 1]
```diff
@@ -4,28 +4,16 @@ These steps will help you provision a Deis cluster.
 
 ## Check System Requirements
 
-Please refer to the [system requirements][] for resource considerations when choosing a machine size to run Deis.
+Please refer to the prequisites and [system requirements][] for considerations when planning your Kubernetes environment.
```
Comment (mboersma) [inline src/installing-deis/quickstart.md]: Typo: "prerequisites"

File: src/installing-deis/system-requirements.md [hunk 2]
```diff
@@ -13,8 +13,34 @@ Note that these estimates are for Deis and Kubernetes only, and there should be
...
+# Kubernetes Requirements
 
-	EXTRA_DOCKER_OPTS="--insecure-registry 10.0.0.0/8"
+## Versions
+
+Deis workflow has been tested with the Kubernetes v1.1 release line. While Kubernetes 1.2 may work we haven't fully tested that release.
+
+## Daemon Sets
+
+The logging components require Kubernetes Daemon Sets API. DaemonSets are not enabled by default in the v1.1 release line, to enable these extensions follow the instructions found [here](http://kubernetes.io/v1.1/docs/api.html#enabling-resources-in-the-extensions-group). If you are running Kubernetes v1.2, DaemonSets are enabled by default.
```
Comment (mboersma) [inline src/installing-deis/system-requirements.md]: "To enable" should start a new sentence.

File: src/installing-deis/system-requirements.md [hunk 3]
```diff
@@ -13,8 +13,34 @@ Note that these estimates are for Deis and Kubernetes only, and there should be
...
+
+Specific steps to enable api extensions may vary based on your Kubernetes configuration. For example, to update a CoreOS Kubernetes cluster edit the API server unit file and add the following line to the `ExecStart` stanza: `--runtime_config=extensions/v1beta1=true,extensions/v1beta1/daemonsets=true`.
```
Comment (mboersma) [inline src/installing-deis/system-requirements.md]: There should be a comma after "cluster": "to update a CoreOS Kubernetes cluster, edit the API...".

File: src/installing-deis/system-requirements.md [hunk 4]
```diff
@@ -13,8 +13,34 @@ Note that these estimates are for Deis and Kubernetes only, and there should be
...
+
+# Docker Requirements
+
+## Docker Version
+
+Any Kubernetes 1.1 cluster should also use a Docker version < 1.10.0 so that `kubectl exec` and Deis database health checks work properly (Refs: [fsouza/go-dockerclient#455](https://github.com/fsouza/go-dockerclient/issues/455) and [kubernetes/kubernetes#19720](https://github.com/kubernetes/kubernetes/issues/19720)).
+
+## Docker Insecure Registry
+
+The on-cluster, Deis managed Docker registry is not deployed with TLS by default. As such, all docker daemons on your Kubernetes worker nodes must be configured with an appropriate "insecure-registry". The subnet given to "insecure-registry" should encompase any private networks used by your hosts including the configured overlay networks. Depending on your Kubernetes configuration `10.0.0.0/8` may be sufficient.
```
Comment (mboersma) [inline src/installing-deis/system-requirements.md]: "Deis-managed" should be a hyphenated phrase.

**Comment by jchauncey** [comment]
The content looks good. I'll let someone else comment on the grammar.

File: src/installing-deis/system-requirements.md (hunk 4 above)
Comment (mboersma) [inline src/installing-deis/system-requirements.md]: I think this should be a bit more explicit that this is a Docker flag:
> ...on your Kubernetes worker nodes must be configured with an appropriate `--insecure-registry` flag.

File: src/installing-deis/system-requirements.md (hunk 2 above)
Comment (mboersma) [inline src/installing-deis/system-requirements.md]: We should be consistent with either `Daemon Sets` or `DaemonSets` in this paragraph. I think the latter.

File: src/installing-deis/quickstart.md (hunk 1 above)
Comment (bacongobbler) [inline src/installing-deis/quickstart.md]: why are we referencing the prerequisites when we are removing them? :)

File: src/installing-deis/system-requirements.md (hunk 2 above)
Comment (bacongobbler) [inline src/installing-deis/system-requirements.md]: :+1: for `DaemonSets`. More language-consistent across kubernetes documentation and our own.

File: src/installing-deis/system-requirements.md (hunk 3 above)
Comment (bacongobbler) [inline src/installing-deis/system-requirements.md]: API should also be capitalized for consistency (API is capitalized in line 29)

File: src/installing-deis/system-requirements.md [hunk 5]
```diff
@@ -13,8 +13,34 @@ Note that these estimates are for Deis and Kubernetes only, and there should be
...
+
+Specific steps to enable api extensions may vary based on your Kubernetes configuration. For example, to update a CoreOS Kubernetes cluster edit the API server unit file and add the following line to the `ExecStart` stanza: `--runtime_config=extensions/v1beta1=true,extensions/v1beta1/daemonsets=true`.
+
+Restart your API server and check that the extensions API is enabled by running:
+
+```
+$ kubectl api-versions
+extensions/v1beta1
+v1
+$
```
Comment (bacongobbler) [inline src/installing-deis/system-requirements.md]: we should remove this extra line prompt if there's nothing to run, just so we're consistent with the rest of the documentation.

File: src/installing-deis/system-requirements.md (hunk 4 above)
Comment (bacongobbler) [inline src/installing-deis/system-requirements.md]: either "Deis-managed" or `Deis' [Registry](../understanding-deis/components.md#Registry)` as we do elsewhere in the docs.

File: src/installing-deis/system-requirements.md (hunk 2 above)
Comment (jchauncey) [inline src/installing-deis/system-requirements.md]: according to their official daemon sets doc it is two words - http://kubernetes.io/v1.1/docs/admin/daemons.html

Comment (mboersma) [inline src/installing-deis/system-requirements.md]: Then I recommend `Daemon Sets` and I have always recommended `Daemon Sets`. :smiling_imp:

**Comment by slack** [comment]
Update, PTAL

**Comment by slack** [comment]
Also rebased.

File: src/installing-deis/system-requirements.md [hunk 6]
```diff
@@ -13,8 +13,33 @@ Note that these estimates are for Deis and Kubernetes only, and there should be
...
-	EXTRA_DOCKER_OPTS="--insecure-registry 10.0.0.0/8"
+## Versions
+
+Deis workflow has been tested with the Kubernetes v1.1 release line. While Kubernetes 1.2 may work we haven't fully tested that release.
+
+## Daemon Sets
+
+The logging components require Kubernetes Daemon Sets API. Daemon Sets are not enabled by default in the v1.1 release line. To enable these extensions, follow the instructions found [here](http://kubernetes.io/v1.1/docs/api.html#enabling-resources-in-the-extensions-group). If you are running Kubernetes v1.2, Daemon Sets are enabled by default.
+
+Specific steps to enable API extensions may vary based on your Kubernetes configuration. For example, to update a CoreOS Kubernetes cluster edit the API server unit file and add the following line to the `ExecStart` stanza: `--runtime_config=extensions/v1beta1=true,extensions/v1beta1/daemonsets=true`.
```
Comment (carmstrong) [inline src/installing-deis/system-requirements.md]: Super mega minor nit: `cluster, edit`

**Pull request merged**